### ISSUE TITLE
Split is not a function

### ISSUE THREAD

**Issue by madebynoxc** [eush77/ass-parser#1]
With latest Node.JS (7.10.0) throws the following error:

> TypeError: text.slice(...).split is not a function
    at e:\BTSync\Projects\Web\UnlimitedQuotebook\node_modules\ass-parser\index.js:54:40
    at e:\BTSync\Projects\Web\UnlimitedQuotebook\node_modules\fzip\src\fzip.js:49:21
    at Array.map (native)
    at e:\BTSync\Projects\Web\UnlimitedQuotebook\node_modules\fzip\src\fzip.js:48:53
    at parseAss (e:\BTSync\Projects\Web\UnlimitedQuotebook\node_modules\ass-parser\index.js:49:10)
    at fs.readFile (e:\BTSync\Projects\Web\UnlimitedQuotebook\index.js:11:28)
    at FSReqWrap.readFileAfterClose [as oncomplete] (fs.js:439:3)

At the following line in index.js:
`var lines = text.slice(begin, end).split('\n');`

**Fix**
Managed to fix the issue by changing line to:
`var lines = text.slice(begin, end).toString().split('\n');`

**Comment by eush77** [comment]
Hi, and thanks for bringing this up!

Do you have a test to demonstrate that?

Also, it seems `text` is not a string in your example, which is more of a fault of the user module, because the argument to this module is expected to be a string. Can you confirm this?

**Comment by madebynoxc** [comment]
Oh, you are right, my fault.
I am getting the list of my subs like that:
```js
fs.readFile(path + items[i], (err, data) => {
  var cont = parser(data);
  ...
});
```
Just realized that **fs** in this case will return a `Buffer` and not a `string`.
Sorry, I was sure that this is because of difference in Node versions

**Comment by eush77** [comment]
No problem!

Maybe it's still beneficial to put a `text = text.toString()` in this module, JS being dynamically typed and all. Parsing the result of `fs.readFile` is  not an unreasonable thing to do.

**Comment by eush77** [comment]
I made the changes in 4c673d8 (published in 0.1.2).
